index.src.html

```diff
@@ -849,6 +864,54 @@ <h3 id="workers">Workers</h3>
 
 </section>
 
+<section>
+  <h2 id="premission-prompt">Permission Prompt</h2>
+
+  <em>This section is not normative.</em>
+
+  According to the discutions
+  [[Issue#23](https://github.com/WICG/private-network-access/issues/23)], the
+  <dfn export local-lt="permission prompt">private network access permission
+  prompt</dfn> is introduced to relax mixed-content checks.
+
+  Private network access permission prompt is a <a spec="permissions">permission</a>.
+
+  <h3 id="permission-goals">Goals</h3>
+
+  The overarching goal is to provide a way for private devices which are not able
+  and not welling to directly talk to any public servers including TLS certificate
+  authorities and not able to apply secure contexts to relax mixed-content checks.
+
+  <h3 id="permission-header">Additional CORS Headers</h3>
+
+  The <dfn export http-header>`Private-Network-Access-Name`</dfn> attempt to
+  provide users a human friendly name in the private network access permission
+  prompt.
```

Comment:
This seems like a repeat of the CORS header modification above? Also, we should not define things in non-normative sections.